articles/cloud-storage/cs-how-use-file-browsers.md

```diff
@@ -146,78 +146,64 @@ To install and configure S3 Browser:
+4. Once the information has been added the installer will connect to the bucket and upload the files to the requested folder. Once the script has finished it will give you a URL output that you should copy to your clipboard and keep it safe.
 
-### Firefox — AtmosFox
+5. Enter the generated URL into your browser and when connecting for the first time, the application will prompt for you to specify an account UID and Shared secret for the account that you want to access using the GUI. Once the UID and secret have been entered, you can test the credentials using the test button, and then click OK to add them and access the bucket.
 
-To install and configure AtmosFox:
+    ![Atmos HTML5 Add account interface](images/cs-file-browsers-html5-2.png)
 
-1. Download and install the [AtmosFox extension](https://addons.mozilla.org/en-US/firefox/addon/atmosfox/)
+6. Should you wish to add additional accounts that are located at the same Cloud storage endpoint, they can be added to the GUI using the COG icon in the top right have corner of the GUI.
 
-2. After you complete the installation, in Firefox, select **AtmosFox Explorer** from the **Tools** menu to launch the extension.
+7. You can now:
 
-    > [!NOTE]
-    > You may need to press the **Alt** key to reveal the **Tools** menu.
-
-    ![AtmosFox menu option](images/cs-file-browsers-ext-4.png)
-
-3. To connect to Cloud Storage, in the *Manage Account* dialog box, specify your Cloud Storage credentials, which you can find in the *Storage* section of the UKCloud Portal.
-
-    > [!NOTE]
-    > The UID value is a combination of the subtenant ID (bucket name) and your user ID, joined by a forward slash (`/`): `<subtenant_id>/<user_id>`. Ensure you enter the correct endpoint in the Atmos URL field, as shown in [API endpoints](#api-endpoints).
-
-    ![AtmosFox Manage Accounts dialog box](images/cs-file-browsers-ext-5.png)
+    - Browse the buckets and files stored in your account
 
-4. When you're done, click **Save**, then **Close**.
+    - Use the toolbar buttons to upload files
 
-5. You can now browse the buckets and files stored in your account.
+    - Double-click items to download them
 
-    You'll see two lists of files: your local machine on the left and your Cloud Storage on the right. Use the large red arrows in the centre to transfer files to and from Cloud Storage. Context menus work well on both sides, enabling you to access a variety of functions.
+    - Access other operations for selected items through the toolbar buttons
 
-    ![AtmosFox interface](images/cs-file-browsers-ext-6.png)
+    ![Atmos HTML5 Application interface](images/cs-file-browsers-html5-1.png)
 
 ## Next steps
 
-This guide has shown you how to create a new Cloud Storage user. For more information about how to use the service, see the following articles:
+This guide has shown you how to browse Cloud Storage using an application. For more information about how to use the service, see the following articles:
```

Comment:
Suggest:
This article has shown you how to browse Cloud Storage buckets using various different applications. ...